### PR TITLE
Populate date dimension

### PR DESCRIPTION
**Pull request by andrewgarner**

### CODE REVIEW

File: spec/services/populate_date_dimension.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  describe '#call' do
+    subject { service.call }
+
+    context 'when no dimensions exist for the date period' do
+      before do
+        allow(Dimensions::Date).to receive(:exists?).and_return(false)
+      end
+
+      it 'returns true' do
+        is_expected.to be_truthy
```
Comment (dwhenry) [inline spec/services/populate_date_dimension.rb]: should this be:
```
 is_expected.to be true
```
given the function has the explicit ```.all?``` call at the end?

**Comment by dwhenry** [comment]
:green_apple:

File: spec/services/populate_date_dimension.rb (hunk 1 above)
Comment (andrewgarner) [inline spec/services/populate_date_dimension.rb]: `be_truthy` matches on not nil or false. I think that should be OK?

**Comment by dwhenry** [comment]
:green_apple: Happy with that to merge

**Pull request merged**